cache/cache_utmp.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/Ptt-official-app/go-pttbbs/types"
 )
 
+func GetUTotal() (total int32) {
+	Shm.ReadAt(
```

Comment:
can you change the function name as GetUTMPNumber?
There is UTotal in the SHM.

Reply:
https://github.com/Ptt-official-app/go-pttbbs/blob/main/cache/shm_raw.go#L28

It's actually "Number" in SHMRaw, but the comment says "user total". (copied from c-pttbbs)

Reply:
OK